modules/ROOT/pages/index.adoc

```diff
@@ -29,18 +33,41 @@ You can monitor your APIs' conformance and notify developers to help improve the
 
 include::_partials/api-gov-console-scap-callouts.adoc[]
 
-== Governance in Exchange and Design Center
+== Governance in Exchange, Design Center, and API Manager
+
+API Governance is integrated with <<exchange-overview>>, <<design-center-overview>>, and <<api-manager-overview>>.
+
+[[exchange-overview]]
+=== Exchange
+
+In Exchange:
+
+* Developers can view published API specifications' conformance status details, discover provided rulesets, and publish custom rulesets. 
+* Implementors can view the ruleset that the instance failed to determine how to fix API instance conformance issues.
```

Comment:
This is a little hard to parse.

Reply:
Yes, yikes! I took out some unnecessary information to simplify.